src/main/java/gregtech/api/items/toolitem/IGTTool.java

```diff
@@ -144,9 +144,31 @@ default ItemStack get(Material material) {
         // Set Material
         toolTag.setString(MATERIAL_KEY, material.toString());
 
+        // Grab the definition here because we cannot use getMaxAoEDefinition as it is not initialized yet
+        AoESymmetrical aoeDefinition = getToolStats().getAoEDefinition(stack);
+
         // Set other tool stats (durability)
         ToolProperty toolProperty = material.getProperty(PropertyKey.TOOL);
-        toolTag.setInteger(MAX_DURABILITY_KEY, toolProperty.getToolDurability());
+
+        // Multiply the durability based on the maximum AOE definition of the tool
+        int maxAOEArea = 1;
+        if (aoeDefinition != AoESymmetrical.none()) {
+            maxAOEArea = Math.max(aoeDefinition.column, Math.max(aoeDefinition.row, aoeDefinition.layer));
+        }
+
+        int durability = toolProperty.getToolDurability() * toolProperty.getDurabilityMultiplier();
+
+        // Most Tool Definitions do not set a base durability, which will lead to ignoring the multiplier if present. So apply the multiplier to the material durability if that would happen
```

Comment:
This comment is more confusing then implementation itself. As it talks about state that does not happens anymore because of that if.

Reply:
The comment is explaining why the if statement is present. At this point, no tools have base durability set in their tool definitions. This means that the durability multiplier set in the tool definition would be ignored, because it will always be multiplied by zero.

Therefore, if the base durability is zero, we are instead multiplying the durability multiplier to the durability calculated from the tool materials, so that it is not ignored. If there is a case were the base durability is not zero, we are instead multiplying the durability multiplier to the base durability and adding it to the durability calculated from the tool materials.

Reply:
I get from implementation what it does and when. But that comment is still confusing me.